blogs/2023/11/13/vscode-copilot-smarter.md

```diff
@@ -26,29 +26,29 @@ To work around this, [GitHub Copilot Chat](https://marketplace.visualstudio.com/
 
-Agents can be contributed by the Client or the Service. At GitHub Universe, there was a demo of a service-side agent, a "Docs Agent" in the github.com Chat experience that knows how to search documentation found in repositories (coming soon to VS Code).
+Agents can be contributed by the client or the service. At GitHub Universe, there was a demo of a service-side agent, a "Docs Agent" in the github.com Chat experience that knows how to search documentation found in repositories (coming soon to VS Code).
 
-Client-side agents can be contributed through traditional VS Code extensions. More on that in the [Extensibility](#extensibility) section, but let's take a look at the two agents that are available in VS Code today: `@workspace` and `@vscode`.
+Client-side agents can be contributed through traditional VS Code extensions. More on that in the [Extensibility](#extensibility) section, but let's take a look at two agents that are available in VS Code today: `@workspace` and `@vscode`.
 
 ### @workspace
 
 The `@workspace` agent knows how to gather context about the code in your workspace, can help you navigate it, find relevant classes, files, etc. Imagine you are in the [VS Code repository](https://github.com/microsoft/vscode) and you want to find out more about the service in charge of the current ICodeEditor; you can use the agent like this:
 
 ![@workspace agent answering question about detecting running extensions](workspace-agent-example.png)
 
-We use natural language to ask the `workspace` agent "what service class do I use to get the current ICodeEditor". From there, the agent does the following to get just the right amount of context to send to the LLM:
+Natural language is used to ask the `workspace` agent "what service class do I use to get the current ICodeEditor". From there, the agent does the following to get just the right amount of context to send to the LLM:
 
-* The vscode repo has been indexed by the [GitHub Search Blackbird service]( https://github.blog/2023-02-06-the-technology-behind-githubs-new-code-search). The `@workspace` agent uses this index as a tool to tap into the repository knowledge graph. The `@workspace` agent runs a semantic search that returns relevant code snippets and meta data. GitHub search service has indexed the top 10K GitHub repositories, with plans to add more.
+* The vscode repo has been indexed by the [GitHub Search Blackbird service]( https://github.blog/2023-02-06-the-technology-behind-githubs-new-code-search). The `@workspace` agent uses this index to tap into the repository knowledge graph. The `@workspace` agent runs a semantic search that returns relevant code snippets and meta data. GitHub search service has indexed the top 10K GitHub repositories, with plans to add more.
```

Comment:
Let's keep using "as a tool" here,
Agents are using tools is an AI concept that we want to emphasis here,